symbolic-debuginfo/src/pdb.rs

```diff
@@ -662,19 +662,39 @@ impl<'s> Unit<'s> {
                 None => continue,
             };
 
+            // skip 0-sized line infos
+            let size = line_info.length.map(u64::from);
+            if size == Some(0) {
+                continue;
+            }
+
             let file_info = program.get_file_info(line_info.file_index)?;
 
             lines.push(LineInfo {
                 address: rva,
-                size: line_info.length.map(u64::from),
+                size,
                 file: self.debug_info.file_info(file_info)?,
                 line: line_info.line_start.into(),
             });
         }
         lines.sort_by_key(|line| line.address);
 
-        // TODO: merge line infos that only differ in their `column` information, which we don’t
+        // Merge line infos that only differ in their `column` information, which we don't
         // care about. We only want to output line infos that differ in their file/line.
+        lines.dedup_by(|second, first| {
```

Comment:
`|current, prev|` might also be a fitting naming scheme, making it slightly clearer that the boolean return value is about `current`.